bcache.c

```diff
@@ -70,7 +70,7 @@ static int bcache_path(struct Account *account, const char *mailbox, char *dst,
   }
 
   size_t mailboxlen = mutt_str_strlen(mailbox);
-  len = snprintf(dst, dstlen - 1, "%s/%s%s%s", MessageCachedir, host, NONULL(mailbox),
```

Comment:
Using `dstlen` is correct. The only caller uses `sizeof(bcache->path)`, so this argument is really the number of bytes available in the buffer, not the length of the longest C string that would fit in there.